src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,27 @@
+package lotto.controller;
+
+import lotto.domain.Lotto;
+import lotto.domain.LottoManager;
+import lotto.domain.LottoPrice;
+import lotto.strategy.ManualPickNumberStrategy;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+public class LottoController {
+
+    public static void startLotto() {
+        System.out.println("구입금액을 입력해주세요");
+        int purchaseAmount = InputView.enterNumber();
+
+        LottoManager lottoManager = new LottoManager(
+            LottoPrice.numberOfLottoCanBuy(purchaseAmount));
```

Comment:
대부분 메서드가 실행 후 반환되는 값을 바로 다른 메서드에서 사용하고 있네요.
어떤 값이 반환되는지 메서드에 들어가서 반환 값을 봐야해서 코드 읽기가 쉽지 않은것 같아요.

그리고 메서드 인자 값이 동일한 메서드를 중복 호출하는 케이스도 존재합니다.
이 부분도 반환 값을 받아서 재사용하면 좋을 것 같아요.
```suggestion
        int autoLottoCount = LottoPrice.numberOfLottoCanBuy(purchaseAmount);
        LottoManager lottoManager = new LottoManager(autoLottoCount);
```

Reply:
`LottoManager`가 인스턴스 생성과 동시에 로또를 만들어 초기화하고 있네요.
인스턴스를 초기화 한 뒤 인스턴스에 `makeLottos()` 메시지를 보내어 로또를 반환 받으면 좀 더 읽기 쉬운 코드가 될 것 같아요.
그리고 `lottos` 멤버 변수도 제거해서 구현해보면 좋을 것 같아요.

Reply:
반환값을 받아서 재사용하게 변경했습니다 !
lottos의 멤버변수를 제거하고 LottoController 에서 인자로 넣어줄 수 있게 변경했습니다.
여기가 살짝 헷갈리는 부분인데, 멤버변수로 갖고 있을 객체와 외부에서 주입받을 객체를 어떻게 구분지을 지가 헷갈리는 거 같아요.
저는 인자가 많은 메소드는 좋지 않다고 해서, 해당 인자를 멤버변수로 넣던가, 클래스로 묶는 방식을 고려했었는데 클래스로 묶게 되면, 그 ]클래스에서 다시 멤버변수로 들어가게 된다고 생각이 되서 이렇게 했었던 거 같아요

Reply:
클래스의 멤버 변수는 인스턴스 정보를 표현할 때 적합합니다.
외부에서 주입 받을 객체는 객체지향 프로그래밍에서 `composition`에 대해 찾아보시면 좋을 것 같아요.